### PR TITLE
MOSViz Logo

### PR DESCRIPTION
**Pull request by robelgeda**
Closes: #118 
This PR will enable MOSViz's logo and icon. It also edits the MOSViz entry point to allow the splash screen display. The entry point is modified similar to how CubeViz's entry point was modified to allow more control. (@astrofrog, @brechmos-stsci)

MOSViz icon:
mosviz_icon

MOSViz logo:
mosviz_logo

Loading screen:
mosviz_splash_screen_shot

Mac example:
screen shot 2018-09-11 at 4 51 54 pm

cc: @kassin  @hcferguson

### CODE REVIEW

File: mosviz/cli.py [hunk 1]
```diff
@@ -1,25 +1,97 @@
...
+    hub = data_collection.hub
+
+    splash.set_progress(100)
+
+    session = glue.core.Session(data_collection=data_collection, hub=hub)
+    ga = GlueApplication(session=session)
+    qapp = QtWidgets.QApplication.instance()
+    ga.setWindowTitle('MOSViz v{0}'.format(__version__))
+    qapp.setWindowIcon(QtGui.QIcon(MOSVIZ_ICON_PATH))
+    ga.setWindowIcon(QtGui.QIcon(MOSVIZ_ICON_PATH))
```
Comment (nmearl) [inline mosviz/cli.py]: I'm unfamiliar with this: are the `setWindowIcon` for the `GlueApplication` and the Qt Application instance really separate things?

Comment (robelgeda) [reply mosviz/cli.py]: @nmearl : They are different in this way:
screen shot 2018-09-12 at 10 05 41 am

Comment (nmearl) [reply mosviz/cli.py]: Is `isinstance(application, qapp.__class__)`? E.g. is the `GlueApplication` just a subclass of the `QApplication`?

Comment (nmearl) [reply mosviz/cli.py]: What I am getting at is, does the window icon need to be set in both places explicitly?

Comment (robelgeda) [reply mosviz/cli.py]: It weird but `isinstance(application, qapp.__class__)` is `False`. `GlueApplication` is a `HubListener` subclass. `qapp.setWindowIcon` sets the icon on the MAC Doc and the `ga.setWindowIcon` sets the icon on the window title bar.

Comment (nmearl) [reply mosviz/cli.py]: That is super weird! Thanks for figuring that out.

**Comment by brechmos-stsci** [comment]
All tests passed in travis. Not sure why this is not updating.  Works locally too. Merging.

**Pull request merged**